frontend/src/components/UserProfile.css

```diff
@@ -0,0 +1,81 @@
+*{
```

Comment:
Usually you would have a space between the selector and the opening curly brace, so instead of "*{" you would have "* {"

Reply:
done